internal/cmd/envs.go

```diff
@@ -117,3 +126,85 @@ coder envs --user charlie@coder.com ls -o json \
 		},
 	}
 }
+
+func createEnvCommand() *cobra.Command {
+	var (
+		org      string
+		img      string
+		tag      string
+		services []string
+	)
+
+	cmd := &cobra.Command{
+		Use:    "create [flags] [environment-name]",
+		Short:  "create a new environment.",
+		Args:   cobra.ExactArgs(1),
+		Hidden: true,
+		Long:   "Create a new environment under the active user.",
+		Example: `
+# create a new environment using default resource amounts
+coder envs create \
+	--image id-of-imported-image \
+	my-env-name
+
+See Flags section for default values.
+
+# create a new environment using custom resource amounts
+coder envs create \
+	--cores 4 \
+	--disk 100 \
+	--memory 8 \
+	--image id-of-imported-image \
+	--org id-of-existing-organization \
+	my-env-name
+
+# same command this time using short-hand flags for a more succinct command experience.
+coder envs create \
+	-c 4 \
+	-d 100 \
+	-m 8 \
+	-i id-of-imported-image my-env-name \
+	-o id-of-existing-organization \
+	my-env-name
```

Comment:
This also feels unnecessary and makes the help section far larger than it needs to be.